docs/source/tutorial-beginner.rst

```diff
@@ -204,7 +204,7 @@ Now that you have written your first program, you're ready to write a more compl
 
 ..
 
-  a. ``from cozmo.util import degrees`` - this utility makes it easy to use degrees as a standard of measurement.
+  a. ``from cozmo.util import degrees`` is a new utility that makes it easy to use degrees as a standard of measurement.
```

Comment:
I don't think "new utility" makes sense in the programming context, perhaps "is an Anki provided utility" or just "is a utility"?